### PR TITLE
Add Flow type checker

### PR DESCRIPTION
**Pull request by lpil**
### Motivation

In an attempt to avoid the same problems on RL with unreliable data and
missing edge cases causing crashes I'm adding Flow. :)
### Pre-merge checklist
- [x] Reviews
  - [x] Code review

### CODE REVIEW

**Comment by asavin** [comment]
I'd say this will become relevant once we have Prismic related tickets on the backlog. Probably a good idea, but not yet relevant - the dynamic part of this site might never happen.

**Comment by lpil** [comment]
It'll be hard to reteroactively add this. I think it's better to do sooner than later.

**Comment by lithin** [comment]
Yeah the sooner this is in, the better - takes quite some time to get used to it

**Comment by lpil** [comment]
@AndrewBestbier @zoeabryant Any opinions? :)

**Comment by zoeabryant** [comment]
As @asavin will be maintaining the project going forward, I think it's his call to make.

**Comment by asavin** [comment]
Going to de-scope this for now, let's get back to this after launch next week.

**Comment by JoeStanton** [comment]
Reopened this after discussion yesterday. I've added a couple of basic annotations, and it should largely work out of the box with anyone using Nuclide/Atom.

**Comment by lpil** [comment]
<£

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,10 @@ test: ## Run the tests
 	$(MOCHA)
 
 
+flow: ## Run the type checker
```
Comment (lpil) [inline Makefile]: Add to PHONY

**Pull request merged**